Shared/messages/ObservationReportController.cpp

```diff
@@ -367,13 +367,13 @@ void ObservationReportController::onMouseClicked(QMouseEvent& event)
   SceneView* sceneView = dynamic_cast<SceneView*>(m_geoView);
   if (sceneView)
   {
-    setControlPoint(sceneView->screenToBaseSurface(event.x(), event.y()));
+    setControlPoint(sceneView->screenToBaseSurface(event.pos().x(), event.pos().y()));
```

Comment:
ditto using `position()`. I think that's correct, but the types are different. We need to validate that's correct still.